app/controllers/concerns/load_settings.rb

```diff
@@ -11,12 +11,18 @@ def load_settings
     end
 
     def build_settings_defaults
-      SettingsObject.create(title: 'general', values: {
-        website_title: '',
-        website_tagline: '',
-        website_description: '',
-        website_logo_url: '',
-        website_url: ''
+      SettingsObject.create(title: "general", values: {
+        website_title: "",
+        website_tagline: "",
+        website_description: "",
+        website_logo_url: "",
+        website_url: ""
+      })
+      SettingsObject.create(title: "styles", values: {
+        css: ""
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.